src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayPay = 40;
+  const discountForThreeDays = 20;
+  const discountForSevenDays = 50;
+
+  if (days < 3) {
```

Comment:
3 and 7 still magic numbers

Reply:
never heard about magic numbers, maybe I missed this topic